frontend/src/components/NavbarComponent.vue

```diff
@@ -16,6 +16,9 @@
           <md-list-item @click="goToSettings">
             <md-icon>settings</md-icon>
           </md-list-item>
+          <md-list-item @click="goToDocs">
```

Comment:
we can add a link here to take users to blackswan.rtfd.io

Reply:
Got it!